docs/swarm/swarm-mode.md

```diff
@@ -110,10 +131,18 @@ $ docker swarm join-token --quiet worker
 SWMTKN-1-49nj1cmql0jkz5s954yi3oex3nedyz0fb0xx14ie39trti4wxv-8vxv8rssmk743ojnwacrr2e7c
 ```
 
-Pass the `--rotate` for `swarm join-token` to the token for a worker or manager
+Because the token is the secret to join the swarm you should update it regurlarly. Some options for when to rotate the token include the following:
```

Comment:
Because the tokens are the secrets necessary to join the swarm, you should be careful with them. In particular, checking in this secret to version control is considered a bad practice, since it allows anyone with access to the  the application source-code to add new nodes to the swarm. Manager tokens are specially sensitive, since allowing a particular node to join as a manager means giving it control over the whole cluster.